### PR TITLE
eprover 1.8 to 1.9

### PR DESCRIPTION
**Pull request by arademaker**
upgrading E Prover from 1.8 to 1.9

### CODE REVIEW

File: Library/Formula/eprover.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Eprover < Formula
   desc "Theorem prover for full first-order logic with equality"
-  homepage "https://www4.informatik.tu-muenchen.de/~schulz/E/E.html"
-  url "https://www4.in.tum.de/~schulz/WORK/E_DOWNLOAD/V_1.8/E.tgz"
-  version "1.8"
-  sha256 "636a5353046680f9c960d02d942df0a55af2e3941676df76e3356a334f6e842e"
+  homepage "http://eprover.org"
+  url "http://wwwlehre.dhbw-stuttgart.de/~sschulz/WORK/E_DOWNLOAD/V_1.9/E.tgz"
```
Comment (DomT4) [inline Library/Formula/eprover.rb]: This actually still seems to work with `https://` so let's keep that protocol. The rest is :+1:

**Comment by arademaker** [comment]
In the website page, the link to the source file is HTTP, I am not sure if the HTTPS will be alive in the near future!

**Comment by arademaker** [comment]
see last comment, anyway, I am submitting the fixed version using HTTPS

**Comment by DomT4** [comment]
If the protocol on the URL changes we can always adjust the formulae but it's nice to use SSL/TLS when available.

Merged in https://github.com/Homebrew/homebrew/commit/e3be566db6bc45f3fc9a245adec2606160398fd0. Thank you for your contribution to Homebrew @arademaker; we appreciate it! :smiley_cat:

**Pull request closed without merging**